src/app/settings.ts

```diff
@@ -41,6 +41,7 @@ const Settings = t.type({
   notificationTypes: NotificationTypes,
   notificationTypesAssignments: t.array(NotificationTypesAssignment),
   eventlist: t.record(t.string, t.string),
+  dashboard: t.record(t.string, t.string),
```

Comment:
Ich würde das konkreter typen, statt einfach ein `record` zu machen:

```
dashboard: t.type({
  substitutionsAdminLink: t.string
}),
```

Cool, ich kann dann sonst mein Kalender abonnieren Link Setting auch in diesen Namespace tun...